### ISSUE TITLE
Add Links

### ISSUE THREAD

**Issue by bk-one** [superlistapp/rich_text_editor#31]
**Screenshots:**

![add-link](https://user-images.githubusercontent.com/37401/105196623-1c8b7900-5b3c-11eb-93d3-b8ed10a6d65c.jpg)

![add-link-dialog](https://user-images.githubusercontent.com/37401/105196650-244b1d80-5b3c-11eb-99bd-7445255b9768.jpg)

![added-link](https://user-images.githubusercontent.com/37401/105196723-3462fd00-5b3c-11eb-9dfd-a54b85fbd3df.jpg)

**Comment by matthew-carroll** [comment]
Much like the ticket for changing text alignment with a popup menu, I think this is more of a product ticket than a super_editor ticket.

@salihgueler let's figure out what Superlist needs from super_editor to make this happen on the product side.

**Comment by bk-one** [comment]
is this not a valid ticket then after our last conversation?

**Comment by matthew-carroll** [comment]
Yeah, this makes sense. Ignore my previous comment. We can implement this for the example app.
